bin/pasta_show_maintainers.py

```diff
@@ -86,72 +76,68 @@ def get_small(title, counts):
-            results.append([title, 'Status', 'LoC in list', 'Total LoC', 'Byte count',  'In list(%)', 'Other directories'])
-            for item in counts_filt:
-                if any(all_maintainers.subsystems[item].status):
-                    results.append([item, all_maintainers.subsystems[item].status[0].value,counts_filt[item][0], counts[item][0], counts_filt[item][1], 
-                    round((counts_filt[item][0]/counts[item][0])*100, 2), ', '.join(dirs[item].difference(dirs_filt[item]))]) 
-                else:
-                    results.append([item, '', counts_filt[item][0], counts[item][0], counts_filt[item][1], round((counts_filt[item][0]/counts[item][0])*100, 2)])
-        elif args.filter != None and args.query == 'maintainers':
-            results.append([title, 'LoC in list', 'Total LoC', 'Byte count',  'In list(%)', 'Entries'])
-            for item in counts_filt:
-                results.append([item, counts_filt[item][0], counts[item][0], counts_filt[item][1], round((counts_filt[item][0]/counts[item][0])*100, 2), ', '.join(maintainers_subs[item]) ])
-        elif args.query == 'entries':
-            results.append([title, 'Status', 'LoC', 'Byte count'])
-            for item in counts:
-                if any(all_maintainers.subsystems[item].status):
-                    results.append([item, all_maintainers.subsystems[item].status[0].value, counts[item][0], counts[item][1]]) 
-                else:
-                    results.append([item, '', counts[item][0], counts[item][1]])
-        else:
-            results.append([title, 'LoC', 'Byte count', 'Entries'])
-            for item in counts:
-                results.append([item, counts[item][0], counts[item][1], ', '.join(maintainers_subs[item])])
+        percentage = lambda item : round((counts_filt[item][0]/counts[item][0])*100, 2)
+        status = lambda item : all_maintainers.subsystems[item].status[0].value if any(all_maintainers.subsystems[item].status) else ''
         
+        cases = dict()
+        cases['entries', True] =[lambda item : [item, counts_filt[item][0], counts[item][0], counts_filt[item][1], percentage(item), status(item), ', '.join(dirs[item].difference(dirs_filt[item]))], [args.query.capitalize(), 'LoC in list', 'Total LoC', 'Byte count',  'In list(%)','Status', 'Other directories']]
+        cases['entries', False] =[lambda item : [item, counts[item][0], counts[item][1], status(item)], [args.query.capitalize(), 'LoC', 'Byte count','Status']]
+        cases['maintainers', True] =[lambda item : [item, counts_filt[item][0], counts[item][0], counts_filt[item][1], percentage(item), ', '.join(maintainers_subs[item])], [args.query.capitalize(), 'LoC in list', 'Total LoC', 'Byte count',  'In list(%)', 'Entries']]
+        cases['maintainers', False] =[lambda item : [item, counts[item][0], counts[item][1]], [args.query.capitalize(), 'LoC', 'Byte count']]
+
+        if 'counts_filt' not in locals():
```

Comment:
I think that is a bit too much of meta-programming here. Consider just having one variable above.